meeting-notes/2020-08-12-to-2020-12-16.md

```diff
@@ -0,0 +1,945 @@
+    - [Grzegorz] Suggestion to build a filter that only will consume events relevant to the rules
+    - This is definetly a place we can optimize things (open/close execve)
+    - At startup we could register the relevant filters based on the rules
+    - [Grzergorz] We can use seccomp filters 
+        - With ptrace(2) we do not have access to kernel structs/unions/headers so we reconstruct from `/proc` with any process generating syscall (clone, fork, unshare, etc)
+        - TLDR we can share a tighter set of optimizations with the producer (It sounds like this might be worth it)
+    - [Loris] We can do the optimization on the target using seccomp at startup
+    - [Nova] To share template for proposal with Loris to write down optimization design doc
+    - [leogr] added sort of benchmark tool to the [event-generator](https://github.com/falcosecurity/event-generator#benchmark)
+        - Falco should not be operating ~100% CPU of 1 core 
+        - Falco engine operatings strictly on a single core (at a time)
+        - [leogr] The output can impact drops 
+            - GKE specific drops (hourrly, daily)
+- [Lore] Musl builds
+    - [Nova] Can we add this to the first Fargate call?
+    - [LeoGR] PR open
+        - Still missing get_addr_info_a(); with MUSL
+
+### Closing
+
+- **MC Next Call**: 
+- **Next Release**: 
+
+
+---
+
+# Falco Community Call
+
+[More Information](https://github.com/falcosecurity/community)
+[Zoom Link](https://zoom.us/my/cncffalcoproject)
+[Recording]()
+
+## 2020-08-12
+
+### MC
+
+- Lorenzo Fontana ([@fntlnz](https://github.com/fntlnz))
+
+### Who joined
+
+- Loris Degioanni ([@ldegio](https://github.com/ldegio))
+- Natch Ruengsakulrach ([@natchaphon-r](https://github.com/natchaphon-r))
+- Spencer Krum ([@nibalizer](https://github.com/nibalizer))
+
+
+### This week highlights
+
+
+### Agenda
+
+- [Lore] userspace instrumentation
+    - Userspace example and docs
+    - BCC
+- [Lore] need help updating the drivers, anyone? :)
+- [Lore] Good news! AWS approved 12k USD for us to use for Falco infra [test-infra#159](https://github.com/falcosecurity/test-infra/issues/159)
+  - We have some codes
+  - What happens when the 12k expire (asked this question to AWS)
+  - Migration, when? Who?
+- [nibalizer] Cool updates to the python-client (not merged yet)
+    - examples [falco-logdna](https://github.com/nibalizer/miscelaneous/blob/master/logdna/falco-logdna.py)
+- [nibalizer] Why is the falso unix socket 700 root/root?
+    - does that makes it so every consumer utility must run as root?
+    - Log perms?
+    ```
+    $ ls -l /var/log/falco/falco.log
+    -rw-r----- 1 syslog adm 10024376 Aug 12 15:45 /var/log/falco/falco.log
+    ```
+- [lore] Push the release one week later to let it be better to handle
+
+### Closing
+
+- **MC Next Call**: 
+- **Next Release**: [0.25.0](https://github.com/falcosecurity/falco/milestone/11) on August, 18th - Who are the releasers?
```

Comment:
```suggestion
```

Because we already have this (see https://github.com/falcosecurity/community/blob/master/meeting-notes/2020-08-12.md).